### DISCUSSION TITLE
"getrange" console output

### DISCUSSION THREAD

**Discussion by NRollo** [cesanta/mongoose#1569] [Ask questions, get answers] [answered]
When running the moongoose web server I randomly get console output like this:
```
2022-06-01 08:38:51 2 mongoose.c:1415:getrange   bytes=0-
2022-06-01 08:38:51 2 mongoose.c:1415:getrange   bytes=2735-372967
```
Looking at line 1415 in the mongoose.c code dosn't give me much information on what this means, - is it at debug information that has wrongly been set as INFO or something I should worried about? 

The platform is esp-idf (ESP32) with Mongoose release 7.6

**Comment by cpq** [comment, marked as answer]
@NRollo thank you. 

Yes, that is a debug leftover: https://github.com/cesanta/mongoose/blob/7.6/mongoose.c#L1415

Please update to 7.7, or to the HEAD.
